src/main/resources/com/movie_collection/css/base.css

```diff
@@ -1,2 +1,224 @@
 @import "bootstrap3.css";
+@import url("https://fonts.googleapis.com/css2?family=Caveat&family=ZCOOL+KuaiLe&display=swap");
+
+/*fill with right colors*/
+
+/*@font-face {font-family: "Graphique W01 Regular";*/
+/*    src: url("ab83b275830b2d585ea027c47e3b237a.eot"); !* IE9*!*/
+/*    src: url("ab83b275830b2d585ea027c47e3b237a.eot?#iefix") format("embedded-opentype"), !* IE6-IE8 *!*/
+/*    url("ab83b275830b2d585ea027c47e3b237a.woff2") format("woff2"), !* chrome、firefox *!*/
+/*    url("ab83b275830b2d585ea027c47e3b237a.woff") format("woff"), !* chrome、firefox *!*/
+/*    url("ab83b275830b2d585ea027c47e3b237a.ttf") format("truetype"), !* chrome、firefox、opera、Safari, Android, iOS 4.2+*!*/
+/*    url("ab83b275830b2d585ea027c47e3b237a.svg#Graphique W01 Regular") format("svg"); !* iOS 4.1- *!*/
+/*}*/
+
+.fontDesign{
+    -fx-font-family: 'Caveat', cursive;
+    -fx-font-size: 35px;
+}
+
+
+* {
+ -fx-nord0: #192C35;
+ -fx-nord1: #4FC4F4;
+ -fx-nord2: #434C5E;
+ -fx-nord3: #4C566A;
+ -fx-nord4: #D8DEE9;
+ -fx-nord5: #E5E9F0;
+ -fx-nord6: #ECEFF4;
+ -fx-nord7: #8FBCBB;
+ -fx-nord8: #88C0D0;
+ -fx-nord9: #81A1C1;
+ -fx-nord10:#5E81AC;
+}
+
+/* NAVBAR */
+.nav-bar{
+    -fx-background-color: -fx-nord0;
+    -fx-border-color: black;
+    -fx-border-width: 0 0 1 0;
+    -fx-border-style: solid;
+}
+
+.nav-bar-search{
+    -fx-background-radius:15px;
+    -fx-border-radius: 15px;
+}
+
+.nav-bar-spinner{
+    -fx-border-color: black;
+    -fx-border-width: 1 1 1 1;
+    -fx-border-style: solid;
+}
+
+.nav-bar-spinner-button{
+    -fx-cursor: hand;
+    -fx-background-color: transparent;
+    -fx-text-fill: white;
+    -fx-border-color: -fx-nord1;
+    -fx-border-width: 1 1 1 1;
+    -fx-border-style: solid;
+}
+
+.nav-bar-spinner-button:hover{
+    -fx-background-color: -fx-nord1;
+    -fx-text-fill: black;
+    -fx-border-color: -fx-nord1;
+    -fx-border-width: 1 1 1 1;
+    -fx-border-style: solid;
+    -fx-cursor: hand;
+}
+
+/* DRAWER  */
+.drawer{
+    -fx-background-color: -fx-nord0;
+}
+
+.menu-bar-top-buttons{
```

Comment:
different icons for all movies and selecting file

Reply:
done 🐱‍💻